remote-method/index.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+
+var workspace = require('loopback-workspace');
+var ModelProperty = workspace.models.ModelProperty;
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var accepts = [];
+var returns = [];
+
+module.exports = yeoman.generators.Base.extend({
+  // NOTE(bajtos)
```

Comment:
I think we should come up with a convention for targeting people in comments: ie @bajtos like GH to make it obvious.